flocker/node/testtools.py

```diff
@@ -298,7 +308,12 @@ def assert_calculated_changes_for_deployer(
     :param set additional_node_config: A set of ``Node`` for other nodes.
     :param expected_changes: The ``IStateChange`` expected to be returned.
     :param Leases leases: Currently configured leases. By default none exist.
+    :param ILocalState local_state: The local_state to pass into
+        calculate_changes.  If None, an empty ``FullySharedLocalState`` will be
```

Comment:
Not a great a default, as mentioned above. Use `FullySharedLocalState(state_changes=[node_state])` or whatever it is as the default I think?